stdlib/public/Concurrency/Actor.swift

```diff
@@ -53,7 +53,7 @@ public func _defaultActorDestroy(_ actor: AnyObject)
 internal func _enqueueOnMain(_ job: UnownedJob)
 
 /// A singleton actor whose executor is equivalent to 
-/// \c DispatchQueue.main, which is the main dispatch queue.
+/// DispatchQueue.main, which is the main dispatch queue.
```

Comment:
```suggestion
/// `DispatchQueue.main`, which is the main dispatch queue.
```

Reply:
Now that one _is_ in the abstract, so it shouldn't take code voice.

Reply:
Writing a code symbol in the abstract without putting it in code voice still doesn't follow reference style.  How about: